operating/create-use-collections.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="create-use-collections"]
+= Creating and using object collections
+include::modules/common-attributes.adoc[]
+:context: create-use-collections
+
```

Comment:
I'd recommend moving the "important" note to the beginning of the section and saying something like the following text (in place of the existing important note above prereqs):

Important: Object collections are only available to RHACS customers using the PostgreSQL database. RHACS Cloud Service uses the PostgresSQL database by default. Other RHACS customers can migrate to the PostgreSQL database with help from Red Hat. PostgreSQL database is available as a Field Trial release. A Field Trial provides approved customers access to Red Hat Advanced Cluster Security Cloud Service for trial purposes. For more information, contact [Red Hat Sales](https://www.redhat.com/en/contact).